Sources/PackageDescription/PackageDescriptionSerialization.swift

```diff
@@ -429,6 +495,16 @@ extension SwiftVersion: Encodable {
 }
 
 extension Version: Encodable {
+    /// Encodes this value into the given encoder.
+    ///
+    /// If the value fails to encode anything, `encoder` will encode an empty
+    /// keyed container in its place.
```

Comment:
I'm not sure if this description is correct.

Reply:
I think this whole file is also an internal implementation details, i.e. manifests aren't expected to serialize anything.  If this is documented, the impression is that manifests should do that.  It's just that I don't think there is a way to hide the public codability conformance from the API.